jobs/rocoto/awips.sh

```diff
@@ -73,7 +78,7 @@ for fhr in $fhrlst; do
     fhmax=84
     if [ $fhr -ge $fhmin -a $fhr -le $fhmax ] ; then
 	if [[ $(expr $fhr % 3) -eq 0 ]]; then
-            fhr3=$(printf %03i $fhr)
+            fhr3=$(printf %03d $((10#$fhr)))
```

Comment:
why is the one on the left wrong?

Reply:
bash has an annoying tendency to treat zero-padded numbers as octals. The 10# forces it to be treated as a decimal (which is then zero-padded again; it's annoying). Without this is will complain about numbers like '018', because 8 is an invalid octal digit.

Reply:
What are the incoming value of fhr?  Does it have the `0` padded?
If it doesn't, you would not get the error.

Reply:
Probably not? Something to evaluate later.

Reply:
Ok.